src/tools/crawler/main.rs

```diff
@@ -31,10 +36,21 @@ const SUMMARY_LOOP_INTERVAL: u64 = 60;
 #[derive(Parser)]
 #[clap(author, version, about, long_about = None)]
 struct Args {
-    #[clap(short, long, value_parser, min_values = 1)]
+    /// The initial addresses to connect to
+    #[clap(short, long, value_parser, min_values = 1, required = true)]
     seed_addrs: Vec<String>,
+
+    /// The main crawling loop interval in seconds
     #[clap(short, long, value_parser, default_value_t = MAIN_LOOP_INTERVAL)]
     crawl_interval: u64,
+
+    /// If present, start an RPC server
+    #[clap(short, long, value_parser)]
+    rpc: bool,
+
+    /// The address to bind the RPC server to
+    #[clap(short, long, value_parser)]
+    bind_rpc: Option<String>,
```

Comment:
why not just have a single `rpc_addr` param that's `Option`al and determines if the RPC server should be started? also, can't the address be a `SocketAddr`, i.e. parsed at `clap` level?